cli/src/commands/init.rs

```diff
@@ -101,6 +103,17 @@ pub(crate) fn cmd_init(
         }
         print_trackable_remote_branches(ui, workspace_command.repo().view())?;
     } else if args.git {
+        if wc_path.join(".git").exists() {
+            return Err(user_error_with_hint(
+                "Did not create a jj repo because there is an existing Git repo in this directory.",
+                format!(
+                    "To create a repo backed by the existing Git repo, run `jj init \
+                     --git-repo={}` instead.",
```

Comment:
Use a raw string here.

Reply:
@PhilipMetzger to avoid the `\-`? Fine by me; I was repurposing and tweaking the existing string or likely would have done that in the first place. Will push an update momentarily.

Reply:
> @PhilipMetzger to avoid the `\-`?

Yes, I'd prefer if we avoided `\` for newlines and used raw strings instead. 

> Fine by me; I was repurposing and tweaking the existing string or likely would have done that in the first place. Will push an update momentarily.

That also makes sense.

Reply:
Ah, I see: it *was* done without the `\` before, and `cargo +nightly fmt --all -- --check` failed, but passes with raw strings. Cool. Resolved!